projects/canopy/src/lib/variant/variant.stories.ts

```diff
@@ -28,6 +28,11 @@ const variants = ['generic', 'info', 'success', 'warning', 'error'];
         <button lg-button variant="outline-primary" lgMarginBottom="none">
```

Comment:
```suggestion
        <button lg-button variant="outline-primary">
```
I know it's a draft but eventually you could do this instead of the `<br />`

Reply:
Ah yes, was meaning to do that :) I still one of the `<br />` though as the buttons display inline by default. That's updated now.